modules/500-dashboard/template_tests/copy_custom_certificate_test.go

```diff
@@ -59,6 +49,7 @@ internal:
 	Context("Default", func() {
 		BeforeEach(func() {
 			f.ValuesSetFromYaml("global", globalValues)
+			f.ValuesSet("global.modulesImages", GetModulesImages())
```

Comment:
don't need here

Reply:
done

Reply:
reverted due to failed test
https://github.com/deckhouse/deckhouse-test-1/actions/runs/3154314187/jobs/5131887278